public/components/custom_panels/panel_modules/visualization_container/visualization_container.scss

```diff
@@ -34,6 +35,43 @@
   text-align: center;
 }
 
+.metricVis {
+  #metricsEditInline {
+    /* for ... reasons (?), margin here causes
+       other rules to activate... providing
+       a more-correct display.
+     */
+    margin: 0 0;
+  }
+
+
+  #aggregation__field {
+    .euiFormLabel {
+      width: 130px
+    }
+  }
+
+  //  #explorerPlotComponent {
+  //    //overflow: hidden;
+  //  }
+  //
+  .svg-container {
+    //height: 425px !important;
+    margin-top: -25px !important;
```

Comment:
Any way to clear the !important? We [had a bug](https://github.com/opensearch-project/dashboards-observability/issues/1263) about this recently.